src/z3c/schema/baseurl/__init__.py

```diff
@@ -11,10 +11,14 @@
 # FOR A PARTICULAR PURPOSE.
 #
 ##############################################################################
-"""
-$Id$
-"""
-
-from z3c.schema.baseurl.interfaces import *
```

Comment:
I approve of removing * imports, and I love the addition of `__all__` 😄 

I have a small worry that this (and similar changes) could be an API breakage, but I haven't tried to verify.

Reply:
I added all contents of the module besides the imports. So at least the tests do not break.